docs/admin/CertificateTransparency.adoc

```diff
@@ -0,0 +1,114 @@
+ca.certTransparency.log.2.enable=true
+ca.certTransparency.log.2.pubKey=MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEKATl2B3SAbxyzGOfNRB+AytNTG<snip>
+ca.certTransparency.log.2.url=http://ct.googleapis.com:80/logs/crucible/
+ca.certTransparency.log.2.version=1
+ca.certTransparency.log.3.enable=false
+ca.certTransparency.log.3.pubKey=MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEiKfWtuoWCPMEzSKySjMjXpo38W<snip>
+ca.certTransparency.log.3.url=http://ct.googleapis.com:80/logs/solera2020/
+ca.certTransparency.log.3.version=1
+ca.certTransparency.log.num=3
+
+. Generate a CSR: e.g.
+.. _PKCS10Client -d . -p passwd -l 2048 -n "cn=ladycfu.test.redhat.com,OU=ladycfu-TEST,O=TestDomain" -o pkcs10-TLS.req_
+. Submit the CSR to an enrollment profile depending on the CT mode in CS.cfg:
+..  if (ca.certTransparency.mode=enabled),
+        use any enrollment profile
+..  if (ca.certTransparency.mode=perProfile),
+        using one of the CT profiles: e.g.
+            caServerCertWithSCT
+. Verify the SCT extension using openssl:
+.. Copy the issued b64 cert into a file. E.g. ct1.pem
+.. Convert the pem to binary:
+... _AtoB ct1.pem ct1.bin_
+.. Display DER certificate content
+... _openssl x509 -noout -text -inform der -in ct1.bin_
+... Observe that the SCT extension is present. E.g.
+
+[literal]
+           CT Precertificate SCTs:  
+               Signed Certificate Timestamp:
+                   Version   : v1 (0x0)
+                   Log ID    : B0:CC:83:E5:A5:F9:7D:6B:AF:7C:09:CC:28:49:04:87:
+                               2A:C7:E8:8B:13:2C:63:50:B7:C6:FD:26:E1:6C:6C:77
+                   Timestamp : Jun 11 23:07:14.146 2020 GMT
+                   Extensions: none
+                   Signature : ecdsa-with-SHA256
+                               30:44:02:20:6E:E7:DC:D6:6B:A6:43:E3:BB:8E:1D:28:
+                               63:C6:6B:03:43:4E:7A:90:0F:D6:2B:E8:ED:55:1D:5F:
+                               86:0C:5A:CE:02:20:53:EB:75:FA:75:54:9C:9F:D3:7A:
+                               D4:E7:C6:6C:9B:33:2A:75:D8:AB:DE:7D:B9:FA:2B:19:
+                               56:22:BB:EF:19:AD
+               Signed Certificate Timestamp:
+                   Version   : v1 (0x0)
+                   Log ID    : C3:BF:03:A7:E1:CA:88:41:C6:07:BA:E3:FF:42:70:FC:
+                               A5:EC:45:B1:86:EB:BE:4E:2C:F3:FC:77:86:30:F5:F6
+                   Timestamp : Jun 11 23:07:14.516 2020 GMT
+                   Extensions: none
+                   Signature : ecdsa-with-SHA256
+                               30:44:02:20:4A:C9:4D:EF:64:02:A7:69:FF:34:4E:41:
+                               F4:87:E1:6D:67:B9:07:14:E6:01:47:C2:0A:72:88:7A:
+                               A9:C3:9C:90:02:20:31:26:15:75:60:1E:E2:C0:A3:C2:
+                               ED:CF:22:A0:3B:A4:10:86:D1:C1:A3:7F:68:CC:1A:DD:
+                               6A:5E:10:B2:F1:8F
+
+.. Alternatively, run asn1 dump like this:
```

Comment:
This line is rendered as:
```
a. Alternatively, run asn1 dump like this:
```
IIUC it's supposed to be (the numbering is broken):
```
iv. Alternatively, run asn1 dump like this:
```
I'd suggest to minimize the use of ordered/unordered lists. For example, the top level bullet points can be replaced with regular text:
```
First, generate a CSR ...
  ...
Then submit the CSR to an enrollment ...
  ...
```